resources/glossary/stochasticity.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Stochasticity
+description: Stochasticity is a measure of the randomness of a process. A highly stochastic process (often shortened to a ‘stochastic process’) – is one that features variables or parameters with different probability distributions. Because of this multiple iterations of the process can return different results. Within the field of simulation, It’s important to differentiate stochastic methods […]
```

Comment:
@ghabs Can you suggest a new abstract for the stochasticity page that fits on two lines (as our existing ones at [hash.ai/wiki](https://hash.ai/wiki) do)? This new page is missing one.

Reply:
Stochasticity is a measure of randomness. The state of a stochastic system can be modeled but not precisely predicted.